definitions.yaml

```diff
@@ -166,6 +180,25 @@ definitions:
         items:
           description: Name of the availability zone
           type: string
+      scaling:
+        type: object
+        description: |
+          Attributes specific to cluster node scaling. To have full control of
+          cluster scaling, min and max can be set to the same value.
+        properties:
+          current:
```

Comment:
I did the same for the `V4ClusterDetailsResponse` now. Here I also added a `current` field. WDYT?

Reply:
@xh3b4sd the plan was to start separating desired and actual state better in our api. 

Please see: https://github.com/giantswarm/giantswarm/pull/2206/files#diff-c512236ee39b9619d47c35ed63f371a4R51

Reply:
I was not aware of that. Thank you for pointing this out. The new status endpoint sounds good, but is like a new feature to me. Do we want to sort this in a separate story or is this what we have to do now? Just saying, this increases complexity and makes the story take longer to be addressed.

Reply:
it is part of the autoscaling story. we need to provide the current number of nodes and therefore we anyway have to read the status. so the idea is to expose all information we have in the status resource. but happy to have a chat about it. hangout might be easier.

Reply:
@xh3b4sd and I decided that the status goes into a separate PR.